cmd/bootstrap/README.md

```diff
@@ -68,7 +68,7 @@ the runtime files generated during initialization.
 Namespaces are stored under ${XDG_DATA_HOME}/.local/share/skupper/namespaces
```

Comment:
I think this should be `${XDG_DATA_HOME}/skupper/namespaces`, instead (ie, without `.local/share`, which is part of the default `${XDG_DATA_HOME}`).

https://specifications.freedesktop.org/basedir-spec/latest/

> $XDG_DATA_HOME defines the base directory relative to which user-specific data files should be stored. If  $XDG_DATA_HOME is either not set or empty, a default equal to $HOME/.local/share should be used.